src/hacking/building-for-openharmony.md

```diff
@@ -4,189 +4,29 @@
 
-The following snippet can be used as a reference for steps 4-6:
-```commandline
-cd ~/ohos-sdk/linux
-for COMPONENT in "native toolchains ets js previewer" do
-    echo "Extracting component ${COMPONENT}"
-    unzip ${COMPONENT}-*.zip
-    API_VERSION=$(cat ${COMPONENT}/oh-uni-package.json | jq -r '.apiVersion')
-    mkdir -p ${API_VERSION}
-    mv ${COMPONENT} "${API_VERSION}/"
-done
-```
-On windows, it is recommended to use 7zip to unzip the archives, since the windows explorer unzip tool is extremely slow.
-
-[DevEco Studio]: https://developer.huawei.com/consumer/cn/deveco-studio
-[OpenHarmony release notes]: https://gitee.com/openharmony/docs/tree/master/en/release-notes/
-[HarmonyOS NEXT commandline tools]: https://developer.huawei.com/consumer/cn/download/
-
-#### Manual installation of the HarmonyOS NEXT commandline tools
-
-The [HarmonyOS NEXT commandline tools] contain the OpenHarmony SDK and the following additional tools:
-
-- `codelinter` (linter)
-- `hstack` (crash dump stack analysis tool)
-- `hvigor` / `hvigorw` (build tool)
-- `ohpm` (package manager)
-
-Currently, the commandline tools package is not publicly available and requires a chinese Huawei account to download.
-
-#### Manual installation of hvigor without the commandline tools
-
-<div class="warning _note">
-This section is not fully tested and may change based on user feedback.
-It's recommended to install the commandline-tools bundle. If you decide to install manually, you need to take
-care to install the hvigor version matching the requirements of your project.
-</div>
-
-`hvigor` (not the wrapper `hvigorw`) is also available via `npm`.
-1. Install the same nodejs version as the commandline-tools ship.
-   For HarmonyOS NEXT Node 18 is shipped. Ensure that the `node` binary is in PATH.
-2. Install Java using the recommended installation method for your OS.
-   The build steps are known to work with OpenJDK v17, v21 and v23.
-   On MacOS, if you install Homebrew's [OpenJDK formula], the following additional command may need to be run after the installation:
-   ```
-   # For the system Java wrappers to find this JDK, symlink it with
-   sudo ln -sfn $HOMEBREW_PREFIX/opt/openjdk/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk.jdk
-   ```
-2. Edit your `.npmrc` to contain the following line:
-
-    ```
-    @ohos:registry=https://repo.harmonyos.com/npm/
-    ```
-
-3. Install hvigor and the hvigor-ohos-plugin. This will create a `node_modules` directory in the current directory.
-   ```commandline
-   npm install @ohos/hvigor
-   npm install @ohos/hvigor-ohos-plugin
-   ```
-4. Now you should be able to run `hvigor.js` in your OpenHarmony project to build a hap bundle:
-   ```
-   /path/to/node_modules/@ohos/hvigor/bin/hvigor.js assembleHap
-   ```
-[OpenJDK formula]: https://formulae.brew.sh/formula/openjdk#default
-
-### Configuring hdc on Linux
```

Comment:
this section is also definitely necessary if you want to use hdc on Linux. We could move it into a separate file though.